_posts/2018-01-02-this-is-not-a-test.md

```diff
@@ -0,0 +1,38 @@
+
+We care about finding team members interested in joining a blameless culture rooting in learning.
+We do this mainly with code review and pairing.
+There are lots of great engineers out there, all with their own styles and preferred ways of working in a team.
+We really wanted to focus on finding team members who would be strengthen the values we have in place, and bring in points of view we may not have on the team yet.
+
+We talked a lot about how we might tailor an interview solution that fits our team, and we came together on on pair programming as that solution. We do not practice strict formal pairing here as a whole,
+but we really like to collaborate, especially on large features.
+It is common for each person to either hop on a call with another engineer about a challenge, an idea, or to talk over code review. Settling on that as the common factor, being able to communicate well with any new person was really key.
+
+In the end we set up a process for pairing with a candidate who could be done in about an hour, and had both a back-end and front-end section (we are a full-stack team).
+We choose to go with a three-person call, one teammate leading the frontend, and another teammate to lead the backend portion. This worked out well for us because then two engineers get to talk to the candidate, and the candidate also gets to ask questions and see how we work with each other.
+
+The idea is to give a small window into how we work, while also getting an idea about how the candidate might fit. Our setup is based on our own codebases, with a few features removed. This worked for us because it was nice to see a candidate work with code that is based on our actual codebase, and also makes it  simpler for us to guide a candidate.
+
+Once we start the exercise with the candidate, the main point is to have them be the ones driving the solutions and communicating with us about what they are thinking. We try to make it clear, that we want to see how they work. We try to really stress that it is not a test. We are completely happy with talking it out, looking at docs for syntax and pseudo-coding solutions as a first pass.
+
+Also, it is great to hear about optimizations the candidate might make, or what they may change if going about it as a task with more time to complete. This lets us get a sense for a person’s personality, disposition and problem-solving style. I have enjoyed getting a chance to talk to so many engineers, and see how they communicate. All in all, the goal is to make it as painless for the candidates as possible while giving us all the chance to get to know each other.
+
+This is an energy-intensive process for us for sure, but so far, the opportunity to meet people and showing where they may fit in with the team has been a really a good investment for us. The team is growing, and it feels like it is growing in all the right ways.
```

Comment:
"like it is growing" => I like "we are" over "it is", but it's just a preference.

Go team!